CMakeLists.txt

```diff
@@ -188,6 +188,8 @@ if($ENV{CLION_IDE})
 	add_definitions(-DPX4_DISABLE_GCC_POISON)
 endif()
 
+add_definitions(-DHEATSHRINK_DYNAMIC_ALLOC=0)
```

Comment:
Only set this where we actually need and use it?
Otherwise move to https://github.com/PX4/PX4-Autopilot/blob/206856638dc3ef1f71e1a34c4a0d8fbcf5da3fd5/cmake/px4_add_common_flags.cmake#L194

Reply:
Ok, I moved it to be more local